database/insertData.js

```diff
@@ -442,6 +453,57 @@ const queries = [
                           "Producer"
                         ]
                       }
+                    },
+                    {
+                      code: "S1PB1"
+                      index: 3
+                      title: "Page Break"
+                      elementTypePluginCode: "pageBreak"
+                      category: INFORMATION
+                      parameters: { previousValidityCheck: true }
```

Comment:
Just FYI -- I think we've removed the need for this parameter since we've gone with `linear`/`non-linear` setting for whole application.

Reply:
GReat. I'll remove from all.